packages/node-sdk/src/utils.ts

```diff
@@ -160,3 +160,16 @@ export function hashObject(obj: Record<string, any>): string {
 
   return hash.digest("hex");
 }
+
+export function once<T extends () => ReturnType<T>>(
```

Comment:
Nice to have a test to validate all cases

Reply:
do you mean, it would be nice to have? Aka, please add a test?

Reply:
yes :)

Reply:
don!